aimmo-game-creator/worker_manager.py

```diff
@@ -171,7 +171,7 @@ def __init__(self, *args, **kwargs):
         super(KubernetesWorkerManager, self).__init__(*args, **kwargs)
 
     def _create_game_rc(self, id, environment_variables):
-        environment_variables['SOCKETIO_RESOURCE'] = "game/%s/socket.io" % id
```

Comment:
Pretty sure changing this is wrong - it won't work under the reverse proxy on Kubernetes